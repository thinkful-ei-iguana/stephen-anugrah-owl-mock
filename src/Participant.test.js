import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Participant from './Participant.js';

describe('<Participant />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Participant />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
})

describe('<Participant />', () => {
  it('renders this UI as expected', () => {

    const part = renderer.create(<Participant />).toJSON();

    expect(part).toMatchSnapshot();
  })
})