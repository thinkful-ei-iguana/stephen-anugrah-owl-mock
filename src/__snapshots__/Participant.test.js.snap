// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participant /> renders this UI as expected 1`] = `
<div
  className="Participant"
>
  <img
    alt="participants avatar"
  />
  <h2 />
  <p>
    left the session
  </p>
</div>
`;
